Source/WebKit/Scripts/webkit/tests/TestWithLegacyReceiverMessages.h

```diff
@@ -333,18 +333,18 @@ class GetPlugins {
     static IPC::MessageName asyncMessageReplyName() { return IPC::MessageName::TestWithLegacyReceiver_GetPluginsReply; }
     static constexpr auto callbackThread = WTF::CompletionHandlerCallThread::ConstructionThread;
     using ReplyArguments = std::tuple<Vector<WebCore::PluginInfo>>;
-    explicit GetPlugins(bool refresh)
+    explicit GetPlugins(const bool& refresh)
```

Comment:
This doesn't look right. `bool` parameters should be  passed by value, as before.